lib/livebook_web/live/session_live/persistence_component.ex

```diff
@@ -49,6 +49,7 @@ defmodule LivebookWeb.SessionLive.PersistenceComponent do
     socket =
       socket
       |> assign(assigns)
+      |> assign_new(:hub, fn -> nil end)
```

Comment:
This is specific to the session page and every notebook has a hub, so there's no case where the hub is missing, right?

```suggestion
```